docs/developer-docs/integrations/sns/launching/launch-steps-1proposal.md

```diff
@@ -28,21 +28,22 @@ ls bazel-bin/rs/sns/cli/sns
 ```
 ## Stages
 
-### 1. Dapp developers choose the initial parameters of the SNS for a dapp.
```

Comment:
I think we deliberatley wanted to have the choosing of the parameters as a first step in the launch as this is actually what might take multiple weeks in reality, so I would not recommend to already add NNS as controller before starting this process. Is there any reason why you think it would be important to change this order?